api/src/app.controller.ts

```diff
@@ -3,32 +3,48 @@ import {
   Controller,
   Get,
   HttpException,
+  Param,
   Post,
   Put,
 } from '@nestjs/common';
 import { AppService } from './app.service';
-import { setGameState } from './game';
-import { GameState, Node } from './types';
+import { initializePlayers } from './game';
+import { GameState } from './types';
+import { GameService } from './game.service';
+import { INITIAL_BOARD } from './constants';
 @Controller()
 export class AppController {
-  constructor(private readonly appService: AppService) {}
+  constructor(
+    private readonly appService: AppService,
+    private readonly gameService: GameService,
+  ) {}
 
-  @Post('startgame')
-  startGame(@Body() body): GameState {
-    if (!body.playerNumber) {
-      throw new HttpException('Argument is missing', 500);
-    }
-    return this.appService.startGame(body.playerNumber);
+  @Get('startgame')
+  async createGame(): Promise<GameState> {
```

Comment:
This should probably be a `post` right ?

Reply:
properly done